package.json

```diff
@@ -20,21 +20,15 @@
     "dist"
   ],
   "scripts": {
-    "clean": "rm -rf dist node_modules test-app/node_modules",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
-    "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
-    "lint:devdeps": "addon-dev sync-dev-deps --lint",
-    "scenario:list": "scenario-tester list --files test-app/scenarios.js",
-    "scenario:output": "scenario-tester output --files test-app/scenarios.js --outdir scenario --scenario ",
-    "scenario:run": "cd test-app && qunit scenarios.js --filter ",
-    "start": "npm-run-all --parallel start:*",
-    "start:test-app": "cd test-app && ember serve",
+    "clean": "rm -rf dist node_modules test-app-3.x/node_modules test-app-4.x/node_modules test-app-5.x/node_modules",
+    "lint": "pnpm --filter '*' lint",
+    "lint:fix": "pnpm --filter '*' lint:fix",
+    "start": "concurrently 'pnpm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
+    "start:test-app": "pnpm --filter test-app-5.x start",
     "start:build": "rollup --config --watch",
     "sync-dev-deps": "addon-dev sync-dev-deps",
-    "test": "npm-run-all lint:* test:*",
-    "test:ember": "cd test-app && ember test",
-    "test:ember-compatibility": "cd test-app && qunit scenarios.js",
+    "test": "pnpm --filter '*' test",
```

Comment:
```suggestion
    "test": "pnpm --filter '*' test:ember",
```

Reply:
this is because the default `test` command does too much. we only want to run tests as we have a separate ci job for linting

Reply:
ah true.... added thanks for input